### ISSUE TITLE
Feature: Tagging based on time

### ISSUE THREAD

**Issue by cvladan** [omnivore-app/omnivore#2748]
First and foremost, I'd like to express my gratitude for making this code available at no cost.

Today, while I was sending articles to Omnivore, an idea struck me about improving the automatic tagging algorithm that you will probably introduce in some future versions.

Typically, when researching, I add articles in what's referred to as 'bursts'. Hence, I believe that the content analysis of these articles should be grouped based on the time they were stored in Omnivore.

So, if the system notices that articles are archived within a time span of less than 10 minutes or so, they should receive the same or similar auto-tag, even if the content doesn't explicitly suggest so.

In a nutshell, that's my idea, and I hope I've explained it clearly.

**Comment by jacksonh** [comment]
Thanks for this suggestion, obviously a lot to think about on how we'd implement it, just want to say its not being ignored.

**Comment by nilsherzig** [comment]
Chrome does something like that for its journey history mode. Might be something to look into

**Comment by Podginator** [comment]
Closing due to age, and the self-hosted nature of Omnivore. Done as a batch action. If this issue still exists, please reopen it.
